pkg/webui/console/views/app/index.js

```diff
@@ -58,6 +61,16 @@ import style from './app.styl'
 const GenericNotFound = () => <FullViewErrorInner error={{ statusCode: 404 }} />
 const errorRender = error => <FullViewError error={error} header={<Header />} />
 
+const m = defineMessages({
+  modalTitle: 'Please sign in again',
+  modalMessage: `You were signed out of your account. Please press 'Reload' to log back in the console again.`,
+  buttonMessage: 'Reload',
+})
+
+const reload = () => {
+  window.location.reload()
+}
```

Comment:
I think to keep the app component lean, you should source this out into a separate container component `/console/containers/log-back-in-modal` or something like this.